troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+```
+
+## Bootstrap VM cannot boot up my nodes
+
+This is a usual problem, maybe we add a typo on the install-config.yaml file or just the privileges are not right on my user. In the worst case you have a problem with your network prerequisites regarding routing and accesses to iDrac from Ironic. 
+To check the problems here we have 3 basic containers all regarding ironic:
+    * Ironic-api
+    * Ironic-conductor
+    * Ironic-inspector
+
+The first to enter in action is the conductor which tries to perform the login and execute the action over IPMI, check the logs there in order to see if there is a problem on this side.
+
+```
+# podman logs -f ironic-conductor
+```
+
+The logs will show up, if there are any errors. 
+
+
+We can also follow the deployment process by seeing the `.openshift_install.log` which will be under `install-dir` folder
+
+```
+# ls -latr kni7/
+auth/                                 .openshift_install.log                terraform.baremetal.auto.tfvars.json  terraform.tfvars.json
+metadata.json                         .openshift_install_state.json         terraform.tfstate                     tls/
+```
+
+# Node Related Issues
+
+* Master Node in `NotReady` state.
+
+If the master nodes are in `NotReady` state, make sure the kubelet services on the master is enabled
+
+```
+oc get nodes
+NAME                                         STATUS     ROLES    AGE   VERSION
+master-0.kni7.cloud.lab.eng.bos.redhat.com   NotReady   master   67m   v1.16.2
+master-1.kni7.cloud.lab.eng.bos.redhat.com   Ready      master   67m   v1.16.2
+master-2.kni7.cloud.lab.eng.bos.redhat.com   Ready      master   67m   v1.16.2
+``` 
+Lets check the status of kubelet of `master-0` 
+
+```
+[core@master-0 ~]$ sudo systemctl status kubelet
+● kubelet.service - Kubernetes Kubelet
+   Loaded: loaded (/etc/systemd/system/kubelet.service; enabled; vendor preset: enabled)
+  Drop-In: /etc/systemd/system/kubelet.service.d
+           └─10-default-env.conf, 20-nodenet.conf
+   Active: inactive (dead) since Tue 2020-03-10 16:05:14 UTC; 33s ago
+ Main PID: 2358 (code=exited, status=0/SUCCESS)
+      CPU: 3min 34.752s
+```
+start the kubelet services and check back the status of the node.
+
+```
+sudo systemctl start kubelet
+[core@master-0 ~]$ sudo systemctl status kubelet
+● kubelet.service - Kubernetes Kubelet
+   Loaded: loaded (/etc/systemd/system/kubelet.service; enabled; vendor preset: enabled)
+  Drop-In: /etc/systemd/system/kubelet.service.d
+           └─10-default-env.conf, 20-nodenet.conf
+   Active: active (running) since Tue 2020-03-10 16:07:27 UTC; 4s ago
+
+oc get nodes
+NAME                                         STATUS   ROLES    AGE   VERSION
+master-0.kni7.cloud.lab.eng.bos.redhat.com   Ready    master   80m   v1.16.2
+master-1.kni7.cloud.lab.eng.bos.redhat.com   Ready    master   80m   v1.16.2
+master-2.kni7.cloud.lab.eng.bos.redhat.com   Ready    master   80m   v1.16.2
+``` 
+ 
+
+* NTP out of sync 
```

Comment:
own section with a header, show error of time being messed up or whats happening that u realize this is the issue.

Reply:
Added a separate section and also explained about why is it needed.